### ISSUE TITLE
grouping not working

### ISSUE THREAD

**Issue by smindorf** [gvalkov/tailon-legacy#4]
Hi,
I try to use grouping but it doesn't wok.

Here my configuration file:

``` code
bind: 127.0.0.1:9998      # address and port to bind on
allow-transfers: true   # allow file downloads
relative-root: /tailon/ # web app root path (default: '')
commands: [tail]  # allowed commands

files:
  - '/var/log/syslog'
  - 'apache':
    - '/var/log/apache2/*.log'
```

and here the debug:

``` code
12:50 $ tailon -c /etc/tailon/config.yaml  -d 
[+62.610149][DEBUG] template dir: /usr/local/lib/python2.7/dist-packages/tailon-0.2.2-py2.7.egg/tailon/../templates
[+62.684059][DEBUG] static dir: /usr/local/lib/python2.7/dist-packages/tailon-0.2.2-py2.7.egg/tailon/../assets
[+68.207026][DEBUG] Config:
{'addr': '127.0.0.1',
 'allow-transfers': True,
 'commands': ['tail'],
 'debug': False,
 'files': OrderedDict([('__ungrouped__', ['/var/log/syslog']), ('apache', ['/var/log/apache2/access.log', '/var/log/apache2/error.log', '/var/log/apache2/other_vhosts_access.log'])]),
 'port': 9998,
 'relative-root': '/tailon/'}
[+68.416119][DEBUG] Files:
ungrouped:
 - /var/log/syslog
 - apache:
 - /var/log/apache2/access.log
 - /var/log/apache2/error.log
 - /var/log/apache2/other_vhosts_access.log
[+68.679094][ INFO] Listening on 127.0.0.1:9998

```

In the browserselection I can only see and access the syslog but no apache logs.

Regards,
Soeren

**Comment by gvalkov** [comment]
Hello,

That functionality was there only on paper - sorry if that misled you. I just implemented in 3bbade706b1eb728d302a73cae3a465ab90ebb15. You can upgrade to tailon 0.3.0 if you would like to test it.

Regards,
Georgi

**Comment by smindorf** [comment]
Hello Gregori, 

I tried the 0.3.0 but now I'm unable to use tailon. 

I've copied assets and templates from 0.2.0 to get the webside working but I'm unable to select any logfile. I tried with grouping and without. I only see the wheel but nothing happens. 

In the debug I see my connection not more. 

Regrads, 
Soeren

---- Von: Georgi Valkov notifications@github.com -- Gesendet: 03.11.2014 - 22:48 ----

**Comment by gvalkov** [comment]
Hello Soeren,

I've made a small mistake in deploying the release to pypi - the
`templates/index.html` was not updated in the distribution. Copying it from
HEAD seems to work for me.

I'll fix this when I get home from work. Thanks for testing it out.

Regards,
Georgi

**Comment by gvalkov** [comment]
Fixed in tailon 0.3.1.
